app/services/renderer_methods.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# macros to fetch information about invoked jobs
+module RendererMethods
```

Comment:
Why do we have this file in `services` folder? :thinking:

Reply:
I 'stole' this place from foreman_ansible. Is somewhere better place for this module? I can move it.